packages/channel-client/.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Does this belong?

Reply:
Probably not strictly, but I think it's probably ok (?) - it's just importing the `.vscode` functionality that probably should have set up when the package was created.